habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/util/PoetryUtil.java

```diff
@@ -15,4 +23,24 @@ public final class PoetryUtil {
      * must be installed and available for Habushu to use.
      */
     public static final String POETRY_CORE_VERSION_REQUIREMENT = "^1.6.0";
+
+
+    public static String findCurrentVirtualEnvironmentFullPathForPoetry(String pythonVersion, String pythonPackageAndDependencyManager,
```

Comment:
A: tabs -> spaces for the method, and potentially
```suggestion
    public static String findVirtualEnvironmentAbsolutePath(String pythonVersion, String pythonPackageAndDependencyManager,
```